packages/smart-account/src/configs.ts

```diff
@@ -0,0 +1,60 @@
+import * as common from '@protocolink/common';
+
+export enum SmartAccountId {
+  PORTUS = '1',
```

Comment:
```suggestion
  PORTUS = 'portus',
```
Prefer to give a readable key rather than serial number